src/runtime/config/configuration-qemu-nvidia-gpu.toml.in

```diff
@@ -13,9 +13,8 @@
 
 [hypervisor.qemu]
 path = "@QEMUPATH@"
-kernel = "@KERNELPATH@"
-image = "@IMAGEPATH@"
-# initrd = "@INITRDPATH@"
+kernel = "/opt/kata/share/kata-containers/vmlinuz-nvidia-gpu.container"
```

Comment:
@zvonkok - did you consider making parameters for the update fields in the Makefile and then templating them in like I think most of the our configurations use for things like `KERNELPATH` , `INITRDPATH`, `KERNELPARAMS`?

Reply:
Ah right, was on the TODO list, will update.

Reply:
@stevenhorsman Fixed, PTAL